app/javascript/components/upload/StepTabHeader.js

```diff
@@ -38,14 +38,14 @@ export default function StepTabHeader({
     badgeContent = <span className="badge fa-xs" data-testid={badgeTestId}><FontAwesomeIcon icon={faCircle}/></span>
   }
 
-  return <li className={className}>
-    <div onClick={() => setCurrentStep(step)} className="step-number">
-      {badgeContent}
-    </div>
-    <div>
-      <a className="action link" onClick={() => setCurrentStep(step)}>
+  return <li className={className} role="tab">
+    <button className="unset-background-border-style" onClick={() => setCurrentStep(step)} >
```

Comment:
We often use classes only styling selectors, but conventionally omit `-style`.

```suggestion
    <button className="unset-background-border" onClick={() => setCurrentStep(step)} >
```